crates/polars-python/src/expr/binary.rs

```diff
@@ -40,6 +42,24 @@ impl PyExpr {
         self.inner.clone().binary().base64_encode().into()
     }
 
+    #[cfg(feature = "binary_encoding")]
+    #[allow(clippy::wrong_self_convention)]
+    fn from_buffer(&self, dtype: Wrap<DataType>, kind: &str) -> PyResult<Self> {
+        use pyo3::exceptions::PyValueError;
+
+        let is_little_endian = match kind.to_lowercase().as_str() {
```

Comment:
I don't want many correct things to do the same. Let's only use `little` and `big` as that's what is documented.

Reply:
fixed.